pikaur/i18n.py

```diff
@@ -0,0 +1,11 @@
+_translation = None
+
+
+def load_translation():
```

Comment:
you need to populate it on first call instead of expecting what somebody wil call this function

Reply:
for example you can just call that function right after defining